src/core/Akka.Remote.Tests.MultiNode/NewRemoteActorSpec.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.Remote.TestKit;
+using Akka.TestKit;
+using Xunit;
+
+namespace Akka.Remote.Tests.MultiNode
+{
+    public class NewRemoteActorMultiNodeSpecConfig : MultiNodeConfig
+    {
+        #region Internal actor classes
+
+        public class SomeActor : UntypedActor
+        {
+            protected override void OnReceive(object message)
+            {
+                if(message.Equals("identify"))
+                    Sender.Tell(Self);
+            }
+        }
+
+        public class SomeActorWithParam : UntypedActor
+        {
+            private string _ignored;
+
+            public SomeActorWithParam(string ignored)
+            {
+                _ignored = ignored;
+            }
+
+
+            protected override void OnReceive(object message)
+            {
+                if (message.Equals("identify"))
+                    Sender.Tell(Self);
+            }
+        }
+
+        #endregion
+
+        readonly RoleName _master;
+        public RoleName Master { get { return _master; } }
+        readonly RoleName _slave;
+        public RoleName Slave { get { return _slave; } }
+
+        public NewRemoteActorMultiNodeSpecConfig()
+        {
+            _master = Role("master");
+            _slave = Role("slave");
+
+            CommonConfig =
+                DebugConfig(false)
+                    .WithFallback(ConfigurationFactory.ParseString("akka.remote.log-remote-lifecycle-events = off"));
+
+            DeployOn(Master, @"
+                /service-hello.remote = ""@slave@""
+                /service-hello-null.remote = ""@slave@""
+                /service-hello3.remote = ""@slave@""
+            ");
+
+            DeployOnAll(@"/service-hello2.remote = ""@slave@""");
+        }
+    }
+
+    public class NewRemoteActorSpecNode1 : NewRemoteActorSpec { }
+    public class NewRemoteActorSpecNode2 : NewRemoteActorSpec { }
+
+    public class NewRemoteActorSpec : MultiNodeSpec
+    {
+        private NewRemoteActorMultiNodeSpecConfig _config;
+
+        public NewRemoteActorSpec()
+            : this(new NewRemoteActorMultiNodeSpecConfig())
+        {
+        }
+
+        public NewRemoteActorSpec(NewRemoteActorMultiNodeSpecConfig config) : base(config)
+        {
+            _config = config;
+        }
+
+        protected override int InitialParticipantsValueFactory
+        {
+            get { return Roles.Count; }
+        }
+
+        protected override bool VerifySystemShutdown
+        {
+            get { return true; }
+        }
+
+        [MultiNodeFact]
+        public void NewRemoteActorSpecs()
+        {
+            ANewRemoteActorMustBeLocallyInstantiatedOnARemoteNodeAndBeAbleToCommunicateThroughItsRemoteActorRef();
+            ANewRemoteActorMustBeLocallyInstantiatedOnARemoteNodeWithNullParameterAndBeAbleToCommunicateThroughItsRemoteActorRef();
```

Comment:
@schatekar that's a shortcoming of our design - in the JVM version they're able to make each individual test method its own multinode fact, but those tests are expected to share state and be executed in sequences. I.E. they don't destroy and recreate the environment on each test method. We'd need to change the way XUnit works (to preserve the state between each test in the same test class) and modify our test runner to the do the same.

So I created a separate little stub method (not actually the constructor) here that calls all three of the test methods in succession - it's not ideal but it's workable in the short run.